css.md

```diff
@@ -0,0 +1,32 @@
+# CSS
+
+**Prerequisite knowledge**: basic html elements
+
+## Intro
+CSS (Cascading Style Sheets) is a style sheet language used for describing the presentation of document written in a markup language (like HTML).
+	CSS can improve content accessibility and readability by affecting layout, colors and fonts. Also  provide more flexibility and control in the specification of presentation characteristics.
+	Enable multiple web pages to share formatting by specifying the relevant CSS in a separate .css file, and reduce complexity and repetition in the structural content. 
+	The name cascading comes from the specified priority scheme to determine which style rule applies if more than one rule matches a particular element. 
+	This cascading priority scheme is predictable. [source](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)
+
+## Classification / acceptance questions
+
+### Apprentice aka _Short skillset description, example: "I can branch!", "My First TODO app"_
+	*I know what is a box model, and I know its basic elements (graphical interpretation)
```

Comment:
Take a look at https://github.com/bethinkpl/bit-docs/blob/49aadb54c96f7f34eecea5044053df7e89965fa6/css.md, formatting is wrong.